test/CMakeLists.txt

```diff
@@ -236,13 +155,13 @@ test_headers(migraphx ${CMAKE_SOURCE_DIR}/src/include/migraphx/*.hpp)
 test_headers(migraphx/ref ${CMAKE_SOURCE_DIR}/src/targets/ref/include/migraphx/ref/*.hpp)
 
 if(MIGRAPHX_ENABLE_GPU)
-    test_headers(migraphx/gpu ${CMAKE_SOURCE_DIR}/src/targets/gpu/include/migraphx/gpu/*.hpp)
```

Comment:
Using `cmake_parse_arguments()` for `HEADERS` only makes sense if it accepts multiple values. Otherwise, it makes the keyword optional. It is supposed to be a fixed argument of the function if it is mandatory.